docs/README.md

```diff
@@ -0,0 +1,214 @@
+
+![Desired Caps To ENV Example](./assets/desired-caps-circleci-env.png)
+
+> More information about Desired Capabilities and its parameters can be found in https://docs.kobiton.com/automation-testing/desired-capabilities-usage/
+
+#### 2.3. CircleCI configuration file
+The provided CircleCI configuration file that will be used looks like this:
+
+```yaml
+version: 2
+jobs:
+  build:
+    working_directory: ~/sample
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - run:
+          name: npm-restore-deps
+          command: npm install
+          working_directory: samples/automation-test
+      - run:
+          name: app-auto-testing
+          command: npm run automation-test-script
+          working_directory: samples/automation-test
+```
+
+Below is in-depth explanation of the provided configuration file:
+
+- Here we use CircleCI official NodeJS Docker image as the execution environment
+
+>```yaml
+>docker:
+>    - image: circleci/node:latest
+>```
+
+- Checkout and pull code from GitHub
+
+>```yaml
+>- checkout
+>```
+
+- Install missing dependencies
+
+>```yaml
+>- run:
+>    name: npm-restore-deps
+>    command: npm install
+>    working_directory: samples/automation-test
+>```
+
+- Execute automation test script on Kobiton
+
+>```yaml
+>- run:
+>    name: app-auto-testing
+>    command: npm run automation-test-script
+>    working_directory: samples/automation-test
+>```
+
+> For more information about how to execute automation test(s) on Kobiton, you can visit:
+> - [Kobiton automation testing documentation](https://docs.kobiton.com/automation-testing/automation-testing-with-kobiton/)
+> - [Kobiton sample automation test scripts in other languages](https://github.com/kobiton/samples)
+
+### 3. Automation Test Execution
+- Simply re-initiate the build process on CircleCI and it will execute the automation test script on Kobiton.
+- Your test execution progress can be viewed on CircleCI.
+
+![CircleCI build execution title](./assets/test-title.png)
+
+![CircleCI build execution report](./assets/test-details.png)
+
+- Your test execution progress can also be viewed on Kobiton
+
+![Kobiton test execution progress](./assets/kobiton-test-executing.png)
+
+## B. Test session details
+### 1. Viewing test session details on Kobiton website
+
+Your test session can be viewed on Kobiton website. Follow these steps below
+
+1. Go to https://portal.kobiton.com/sessions, login with your Kobiton account.
+2. You will see your executed sessions and their statuses.
+
+![Kobiton session](./assets/session-kobiton.png)
+
+3. Click on any session to view its details, commands.
+
+### 2. Fetching test session details using Kobiton REST API
+
+Kobiton has already provided samples written in NodeJS to get session information, commands using Kobiton REST API.
+
+Refer to https://github.com/kobiton/samples/tree/master/kobiton-rest-api for instructions.
```

Comment:
`Refer to [Kobiton sample for REST API](https://github.com/kobiton/samples/tree/master/kobiton-rest-api) ...`